src/Files.App/Shell/Win32Shell.cs

```diff
@@ -31,17 +34,21 @@ public static async Task<(ShellFileItem Folder, List<ShellFileItem> Enumerate)>
 			{
 				var flc = new List<ShellFileItem>();
 				var folder = (ShellFileItem)null;
+
 				try
 				{
 					using var shellFolder = ShellFolderExtensions.GetShellItemFromPathOrPidl(path) as ShellFolder;
 					folder = ShellFolderExtensions.GetShellFileItem(shellFolder);
-					if ((controlPanel.PIDL.IsParentOf(shellFolder.PIDL, false) || controlPanelCategoryView.PIDL.IsParentOf(shellFolder.PIDL, false))
-						&& !shellFolder.Any())
+
+					if ((controlPanel.PIDL.IsParentOf(shellFolder.PIDL, false) ||
+						controlPanelCategoryView.PIDL.IsParentOf(shellFolder.PIDL, false)) &&
+						!shellFolder.Any())
 					{
 						// Return null to force open unsupported items in explorer
 						// Only if inside control panel and folder appears empty
```

Comment:
I think comments should be placed before the if
`Only` should begin with lowercase `o`